snomed/com.b2international.snowowl.snomed.exporter.server/src/com/b2international/snowowl/snomed/exporter/server/refset/SnomedSimpleTypeRefSetDSVExporter.java

```diff
@@ -316,16 +315,11 @@ public boolean apply(ISnomedDescription input) {
 								}).transformAndConcat(new Function<ISnomedDescription, List<String>>() {
```

Comment:
You can replace this transformAndConcat with a simple transform and get rid of the singleton lists, since now we are using single items in any case.